swagger/swagger.yaml

```diff
@@ -738,6 +743,39 @@ components:
           $ref: "#/components/schemas/ThreatTypeEnum"
         severity:
           $ref: "#/components/schemas/ThreatSeverityEnum"
+    Event:
+      required:
+        - unstructured
+        - type
+      type: object
+      description: an unit of structured communication from scenario to ADM
```

Comment:
same comment as server.

Reply:
This is addressed in [ITM-477's PR](https://github.com/NextCenturyCorporation/itm-evaluation-client/pull/43).